### DISCUSSION TITLE
Single Player - Show ranking while playing a beatmap

### DISCUSSION THREAD

**Discussion by Morodar** [ppy/osu#14154] [Ideas] [answered]
# Summary

In osu!stable you can select the high-score type which will also be shown while playing the beatmap.
In osu!lazer however, you can also select the high-score type but it will not be shown while playing the map.

# Score type selection
To clarify, I'm referring to this selection:

![image](https://user-images.githubusercontent.com/10113399/128595558-2abdd319-c600-4600-b70e-7faeed09ebcd.png)

# osu!stable
This is what it looks like in stable (when local is selected):

![Animation6](https://user-images.githubusercontent.com/10113399/128595500-764a5a84-39a5-4c62-a5b6-d454dba16151.gif)

**Comment by Morodar** [comment]
I've checked the issues and questions sections (searched for 'score') so I hope this is not a duplicate.

**Reply by NotGumballer91** [reply]
What did you use to record it in GIF like that? And what encoding settings?

**Reply by Morodar** [reply]
I'm using ScreenToGif <https://github.com/NickeManarin/ScreenToGif/>. You have to set osu! to borderless in order to capture it. I use default default settings and try to keep gifs short to reduce file size.

**Comment by peppy** [comment]
The logic to make this happen is already mostly present, and just needs to be hooked up properly. There are plans to change the display style of this leaderboard going forward (showing more than just the top 50), but that required infrastructure changes, so getting this to parity with osu-stable is a good first step.

**Reply by Moltanicaa** [reply]
Do you know the approximate implementation period? Perhaps, this is the last visual gameplay option that is missing during the game.

**Reply by peppy** [reply]
after realm

**Reply by memesnightmares** [reply]
Thinking about this again now that online leaderboards are live, would be cool to see this Soon(tm)

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/18879
